mne/gui/help/kit2fiff.json

```diff
@@ -1,7 +1,10 @@
 {
+  "hsp_file": "*.hsp or *.txt file containing digitized head shape points",
+  "fid_file": "*.elp or *.txt file containing 8 digitized fiducial points (\"electrode laser points\")",
```

Comment:
However looking at that description again, what does ELP really stand for? "Electrode Location Point"?